code/game/machinery/iv_drip.dm

```diff
@@ -72,8 +73,10 @@
 	if(src.attached)
 
 		if(!(get_dist(src, src.attached) <= 1 && isturf(src.attached.loc)))
-			visible_message("The needle is ripped out of [src.attached], doesn't that hurt?")
-			src.attached:apply_damage(3, BRUTE, pick("r_arm", "l_arm"))
+			var/obj/item/organ/external/affecting = src.attached:get_organ(pick("r_arm", "l_arm"))
+			src.attached << "<span class='danger'>The needle <B>painfully</B> rips out of your [affecting.limb_name == "r_arm" ? "right arm" : "left arm"].</span>"
+			oview(src.attached) << "<span class='warning'>The needle is ripped out of [src.attached]'s [affecting.limb_name == "r_arm" ? "right arm" : "left arm"].</span>"

```

Comment:
Change the `src.attached <<` and `oview(src.attached) <<` to `src.attached.visible_message()`.

Args for visible_message go like this: `proc/visible_message(var/observer_message, var/source_message, var/audible_message)` with only the first one being mandatory.